adr/0003-monitor-condition-and-data-selectors.md

```diff
@@ -27,6 +27,8 @@ Layer model:
   -------------      ---------      ------------
 ```
 
+Integrations can set the `entity_registry_enabled_default` property on their entity objects to instruct Home Assistant to disable certain entities by default.
```

Comment:
TBF, I don't understand how this works, but shouldn't this be named `disabled` if it's used to disable conditions? Can we have a more concrete example of how this is used / implemented by integration developers?

Reply:
There are multiple implementations in the code base should we reference integrations using this functionality?

Reply:
The first time that an entity is added to the entity registry, a user can return `False` from this property (default is `True`) and it will add the entity to the registry, but disable it. The user can then later enable it in the entity registry if they care about this data.